lib/cell/i18n.rb

```diff
@@ -0,0 +1,16 @@
+module Cell::I18n
+
+  def t(*args)
+    options   = args.last.is_a?(Hash) ? args.pop.dup : {}
+    key       = args.shift
+    cell_name = self.class.to_s.delete('Cell').underscore
```

Comment:
yeah, that's exactly what i've done locally but forgot to send a PR for that. thanks